models/labels/balancer_v2/labels_balancer_v2_gauges_arbitrum.sql

```diff
@@ -0,0 +1,35 @@
+{{config(alias='balancer_v2_gauges_arbitrum',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["jacektrocinski"]\') }}')}}
+
+SELECT
+    array('arbitrum') AS blockchain,
```

Comment:
we're no longer using `array<string>` as the blockchain column, just a `string` now.